src/piidigger/classes.py

```diff
@@ -6,17 +6,12 @@
 import platform
 import string
 
-# try:
-#     # If we're on Python later than 3.11 then tomllib is included.  We'll use the "toml" namespace so that the same code will work on older Pythons where toml had to be installed.
-#     import tomllib as toml
-# except ModuleNotFoundError:
 import tomli
 
-from piidigger import globalfuncs
 from piidigger import console
+from piidigger import globalfuncs
 from piidigger.getmime import testMagic
-
-logger=logging.getLogger(__name__)
```

Comment:
Module-level logger no longer needed.  Logging instances are added to specific Classes below as needed.